client/components/content-elements/tce-image/edit/index.vue

```diff
@@ -76,7 +95,13 @@ export default {
       if (imageAvailable) return false;
       if (this.$refs.cropper) this.$refs.cropper.destroy();
       return true;
-    }
+    },
+    elementWidth: ({ containerWidth, element }) => get(element, 'data.meta.width'),
+    elementHeight: ({ containerWidth, element }) => get(element, 'data.meta.height'),
+    maxWidth: ({ containerWidth, elementWidth }) =>
+      containerWidth > elementWidth ? elementWidth : containerWidth,
+    aspectRatio: ({ elementHeight, elementWidth }) =>
+      elementHeight && elementWidth && elementWidth / elementHeight
```

Comment:
Consider wrapping last && into ( ) to make it a bit more readable or implement as ternary